### PR TITLE
Auto-fix test

### PR DESCRIPTION
**Pull request by ha17**
Promoting Tim's comment below:

```
Examples of the types of fixes you'll see in the change set:

Brace alignment
Missing namespace declaration
Class files containing more than 1 class
Line length exceeded (max 120 characters)
Unnecessary newlines between braces
```

The main checks are under `./src`, which are the auto and manual fixes to the code styling, to get this repo into firm PSR12 territory.

---

We don't necessary need to use this, but Tim and I wanted to look at what an auto-run would look like.   

Ran: `../coding-standards/vendor/bin/phpcbf --standard=../coding-standards/src/ruleset.xml ./`

### CODE REVIEW

File: src/Attributes/Column.php [hunk 1]
```diff
@@ -1,102 +1,102 @@
...
-     * Returns Converter instance.
-     *
-     * @return ?Converter
-     */
-    public function getConverter(): ?Converter
-    {
-        return $this->converter;
-    }
-}
+<?php
```
Comment (pcrist) [inline src/Attributes/Column.php:1]: Is this tabs vs spaces or something else that changes the entire file?

Comment (ha17) [reply src/Attributes/Column.php:1]: It's difficult to tell but it appears the line endings changed, going from CFLF to LF.

**Comment by timallen527** [comment]
@ha17 Thanks for getting this out there. Any reason to not go ahead and add `pdga/coding-standards` as a dependency and provide a command to run it with the appropriate ruleset?

**Comment by ha17** [comment]
> @ha17 Thanks for getting this out there. Any reason to not go ahead and add `pdga/coding-standards` as a dependency and provide a command to run it with the appropriate ruleset?

I updated composer; there is a `Increment on type bool has no effect` when you do `composer run format`.

**Comment by timallen527** [comment]
These commits include manual fixes that could not be applied automatically:

- [Enforcer fixes.](https://github.com/PDGA/php-data-objects/pull/69/commits/a4a662318e8336bf144be5d20104138415187aae)

- [Model fixes.](https://github.com/PDGA/php-data-objects/pull/69/commits/699fa5bf31cf1cde6d3c30eb97fdd6de84430a2f)

- [Parser fixes.](https://github.com/PDGA/php-data-objects/pull/69/commits/75c8bdd0110d72322da1aa3c6bd7e278ce3918ec)

- [Attributes fixes.](https://github.com/PDGA/php-data-objects/pull/69/commits/fbfd48343de9a198414d730d1369950b2bf9b237)

- [Validator fixes.](https://github.com/PDGA/php-data-objects/pull/69/commits/7bde7a2820da87d873a97ac1fa827ef8a84b7b7e)

**Comment by timallen527** [comment]
Example output from workflow action with a styling error.
https://github.com/PDGA/php-data-objects/actions/runs/9614325538/job/26518879341#step:5:19

image

**Comment by timallen527** [comment]
Examples of the types of fixes you'll see in the change set:
- Brace alignment
- Missing namespace declaration
- Class files containing more than 1 class
- Line length exceeded (max 120 characters)
- Unnecessary newlines between braces

File: src/Converters/Test/YesNoConverterTest.php [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <?php
 
-namespace PDGA\DataObjects\Converters;
+namespace PDGA\DataObjects\Converters\Test;
```
Comment (pcrist) [inline src/Converters/Test/YesNoConverterTest.php:3]: Is this change (to move all the test classes into their own `Test` directory) part of the coding standards?

**Review by pcrist — COMMENTED** [review]
LGTM 👍

**Review by pcrist — APPROVED** [review]

File: src/Converters/Test/YesNoConverterTest.php (hunk 2 above)
Comment (timallen527) [reply src/Converters/Test/YesNoConverterTest.php:3]: Not directly. Some of the test class files had inner classes defined to help with testing. Those inner classes had to be moved to their own file and it didn't seem appropriate to have those test specific classes mixed in with everything else. So, moved those to `Test` namespaces and I also took a liberty here to make it all consistent.

**Review by drew-acheson — APPROVED** [review]

**Review by evankkerns — APPROVED** [review]

**Pull request merged**